# Translations template for nova.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the nova project.
#
# Translators:
# Zbyněk Schwarz <zbynek.schwarz@gmail.com>, 2015
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: nova 14.0.0.0b3.dev750\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-08-25 06:14+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2015-12-18 02:33+0000\n"
"Last-Translator: Zbyněk Schwarz <zbynek.schwarz@gmail.com>\n"
"Language: cs\n"
"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Czech\n"

#, python-format
msgid ""
"%(cell)s has not been seen since %(last_seen)s and is being treated as mute."
msgstr "%(cell)s nebyla zjištěna od %(last_seen)s a je považována za ztišenou."

#, python-format
msgid "%(cmd)s failed. Sleeping %(time)s seconds before retry."
msgstr "%(cmd)s selhalo. Před dalším pokusem se bude se čekat %(time)s vteřin."

#, python-format
msgid "%(host_state)s has not been heard from in a while"
msgstr "%(host_state)s již se nějakou dobu nehlásí"

#, python-format
msgid "%(logprefix)s failed to load json"
msgstr "%(logprefix)s nemohl načíst json"

#, python-format
msgid "%(logprefix)s file does not exist"
msgstr "Soubor %(logprefix)s neexistuje"

#, python-format
msgid "%(logprefix)s unexpected IOError when reading"
msgstr "%(logprefix)s neočekávaný konec soubor při čtění"

#, python-format
msgid "%(overs)s quota exceeded for %(pid)s, tried to resize instance."
msgstr ""
"Překročena kvóta %(overs)s pro %(pid)s, pokus o změnu velikosti instance."

#, python-format
msgid "%(plugin)s.%(fn)s failed. Retrying call."
msgstr "%(plugin)s.%(fn)s selhalo. Opakování volání."

#, python-format
msgid "%s failed. Not Retrying."
msgstr "%s selhalo. Nebude zopakováno."

#, python-format
msgid "%s is already mounted"
msgstr "%s již je připojeno"

#, python-format
msgid "%s swap image was used by instance but no back files existing!"
msgstr ""
"%s odkládací obraz byl použit instanci ale žáfné záložní soubory neexistují!"

#, python-format
msgid "Address |%(address)s| is not allocated"
msgstr "Adresa |%(address)s| není přidělena"

#, python-format
msgid "Address |%(address)s| is not allocated to your project |%(project)s|"
msgstr "Adresa |%(address)s| není přidělena pro Váš projekt |%(project)s|"

#, python-format
msgid ""
"Assigning a pci device without numa affinity toinstance %(instance)s which "
"has numa topology"
msgstr ""
"Přidělování zařízení pci bez slučivosti k numa k instanci %(instance)s, "
"která má topologii numa"

#, python-format
msgid "Attempted auto_configure_disk failed because: %s"
msgstr "Pokus o automatické nastavení disku selhalo z důvodu: %s"

#, python-format
msgid "Attempted to remove chain %s which does not exist"
msgstr "Pokus o odstranění řetězu %s, který neexistuje"

#, python-format
msgid ""
"Cannot add security group %(name)s to %(instance)s since the port "
"%(port_id)s does not meet security requirements"
msgstr ""
"Nelze přidat bezpečnostní skupinu %(name)s do %(instance)s protože port "
"%(port_id)s nesplňuje bezpečnostní požadavky"

#, python-format
msgid "Cannot decode JSON from %(id_path)s"
msgstr "Nelze dešifrovat JSON z %(id_path)s"

#, python-format
msgid "Cannot delete domain |%s|"
msgstr "Nelze smazat doménu |%s|"

#, python-format
msgid "Cannot delete entry |%s|"
msgstr "Nelze smazat položku |%s|"

msgid "Cannot destroy instance, operation time out"
msgstr "Nelze zničit instanci, operaci vypršel časový limit"

msgid ""
"Cannot get the number of cpu, because this function is not implemented for "
"this platform. "
msgstr ""
"Nelze získat počet procesorů, protože tato funkce není pro tuto platformu "
"zavedena."

msgid "Cannot purge SR with referenced VDIs"
msgstr "Nelze smazat SR pomocí odkazovaných VDI"

#, python-format
msgid ""
"Cannot update service status on host \"%s\" due to an unexpected exception."
msgstr ""
"Nelze aktualizovat stav služby na hostiteli \"%s\" kvůli nečekané výjimce."

#, python-format
msgid "Cannot update service status on host \"%s\" since it is not registered."
msgstr ""
"Nelze aktualizovat stav služby z hostitele \"%s\" protože není registrována."

#, python-format
msgid "Change will make usage less than 0 for the following resources: %s"
msgstr "Změna využití zmenší na méně než 0 pro následující zdroje: %s"

msgid ""
"Cinder V1 API is deprecated as of the Juno release, and Nova is still "
"configured to use it. Enable the V2 API in Cinder and set cinder."
"catalog_info in nova.conf to use it."
msgstr ""
"API V1 Cinder je zastaralá od verze Juno a Nova je stále nastavena pro její "
"použití. Povolte API V2 v Cinder a nastavte cinder.catalog_info v nova.conf, "
"abyste ji mohli používat."

msgid ""
"Conductor local mode is deprecated and will be removed in a subsequent "
"release"
msgstr "Místní režim Conductor je zastaralý a v následné verzi bude odstraněn."

#, python-format
msgid "Could not clean up failed build, not rescheduling. Error: %s"
msgstr "Nelze vyčistit sestavení co selhalo, není znovu naplánováno. Chyba: %s"

#, python-format
msgid "Could not decode cpu_allocation_ratio: '%s'"
msgstr "Nelze dešifrovat poměr přidružení procesoru: '%s'"

#, python-format
msgid "Could not decode disk_allocation_ratio: '%s'"
msgstr "Nelze dešifrovat poměr přidružení disku: '%s'"

#, python-format
msgid "Could not decode max_instances_per_host: '%s'"
msgstr "Nelze dešifrovat maximum instancí na hostitele: '%s'"

#, python-format
msgid "Could not decode max_io_ops_per_host: '%s'"
msgstr "Nelze dešifrovat maximum operací vstupu/výstupu na hostitele: '%s'"

#, python-format
msgid "Could not decode ram_allocation_ratio: '%s'"
msgstr "Nelze dešifrovat poměr přidělení ram: '%s'"

msgid "Could not determine iscsi initiator name"
msgstr "Nelze zjistit název zavaděče iscsi"

#, python-format
msgid "Could not determine key: %s"
msgstr "Nelze zjistit klíč: %s"

#, python-format
msgid ""
"Couldn't retrieve the online CPUs due to a Libvirt error: %(error)s with "
"error code: %(error_code)s"
msgstr ""
"Nelze získat procesory, které jsou online, kvůli chybě Libvirt: %(error)s "
"mající kód chyby: %(error_code)s"

#, python-format
msgid ""
"Database inconsistency: DNS domain |%s| is registered in the Nova db but not "
"visible to either the floating or instance DNS driver. It will be ignored."
msgstr ""
"Nesoulad databáze: Doména DNS |%s| je registrována v Nova db, ale není "
"viditelná plovoucímu ovladači DNS ani instančnímu. Bude ignorována."

#, python-format
msgid ""
"Destination %s already exists! Concurrent moves can lead to unexpected "
"results."
msgstr ""
"Cíl %s již existuje! SOuběžné přesuny mohou vést k nečekaným výsledkům."

#, python-format
msgid "Destroy called on non-existing instance %s."
msgstr "Zničení voláno na neexistující instanci %s."

#, python-format
msgid "Detach interface failed, port_id=%(port_id)s, reason: %(msg)s"
msgstr "Odpojení rozhraní selhalo, id portu=%(port_id)s, důvod: %(msg)s"

#, python-format
msgid "Detaching from erroneous nbd device returned error: %s"
msgstr "Odpojování z chybného zařízení nbd vrátilo chybu: %s"

msgid "Detaching volume from unknown instance"
msgstr "Odpojování svazku z neznámé instance"

#, python-format
msgid ""
"Detected instance with name label '%s' which is marked as DELETED but still "
"present on host."
msgstr ""
"Zjištěna instance se jmenovkou '%s', jenž je označená jako SMAZÁNA, ale "
"stále přítomna na hostiteli."

#, python-format
msgid ""
"Detected running orphan instance: %(uuid)s (consuming %(memory_mb)s MB "
"memory)"
msgstr ""
"Zjištěna spuštěná osiřelá instance: %(uuid)s (spotřebovávající %(memory_mb)s "
"MB paměti)"

msgid "Device allocation failed after repeated retries."
msgstr "Přidělení zařízení selhalo po několika pokusech."

#, python-format
msgid "Domain |%(domain)s| already exists, changing project to |%(project)s|."
msgstr "Doména |%(domain)s| již existuje, změna projektu na |%(project)s|."

#, python-format
msgid "Domain |%(domain)s| already exists, changing zone to |%(av_zone)s|."
msgstr "Doména |%(domain)s| již existuje, změna zóny na |%(av_zone)s|."

#, python-format
msgid "Driver failed to detach volume %(volume_id)s at %(mount_point)s."
msgstr "Ovladač nemohl odpojit svazek %(volume_id)s v %(mount_point)s."

msgid "During detach_interface, instance disappeared."
msgstr "Instance zmizela během odpojování rozhraní."

msgid "During detach_volume, instance disappeared."
msgstr "Instance zmizela během odpojování svazku."

msgid "Error cleaning up fixed IP allocation. Manual cleanup may be required."
msgstr ""
"Chyba při čistění přidělení pevných IP adres. Bude zapotřebí je vyčistit "
"ručně."

#, python-format
msgid ""
"Error from libvirt during destroy. Code=%(errcode)s Error=%(e)s; attempt "
"%(attempt)d of 3"
msgstr ""
"Chyba od libvirt během ničení. Kód=%(errcode)s Chyba=%(e)s; pokus "
"%(attempt)d ze 3"

#, python-format
msgid ""
"Error from libvirt while getting description of %(instance_name)s: [Error "
"Code %(error_code)s] %(ex)s"
msgstr ""
"Chyba od libvirt při získávání popisu %(instance_name)s: [Kód chyby "
"%(error_code)s] %(ex)s"

#, python-format
msgid "Error monitoring migration: %(ex)s"
msgstr "Chyba při sledování přesunu: %(ex)s"

#, python-format
msgid "Error while trying to clean up image %s"
msgstr "Chyba při pokusu o vyčištění obrazu %s"

#, python-format
msgid ""
"Excluding %(namespace)s monitor %(monitor_name)s. Already loaded "
"%(loaded_monitor)s."
msgstr ""
"Monitor %(monitor_name)s sledující %(namespace)s vynechán. "
"%(loaded_monitor)s již načten."

#, python-format
msgid ""
"Excluding %(namespace)s monitor %(monitor_name)s. Not in the list of enabled "
"monitors (CONF.compute_monitors)."
msgstr ""
"Monitor %(monitor_name)s sledující %(namespace)s vynechán. Není v seznamu "
"povolených monitorů (CONF.compute_monitors)."

#, python-format
msgid "Expected to receive %(exp)s bytes, but actually %(act)s"
msgstr "Očekáváno přijetí %(exp)s bajtů, ve skutečnosti obdrženo %(act)s"

#, python-format
msgid ""
"Extension %(ext_name)s: Cannot extend resource %(collection)s: No such "
"resource"
msgstr ""
"Rozšíření %(ext_name)s: nelze rozšířit %(collection)s: Žádný takový zdroj"

#, python-format
msgid ""
"Extension %s is both in whitelist and blacklist, blacklisting takes "
"precedence"
msgstr ""
"Rozšíření %s je jak v seznamu povolených tak i zakázaných, zakázání má "
"přednost"

#, python-format
msgid "Failed to %(service)s_%(method)s: %(ex)s"
msgstr "Nelze %(service)s_%(method)s: %(ex)s"

#, python-format
msgid "Failed to abort migration %s"
msgstr "Nelze ukončit přesun %s"

#, python-format
msgid "Failed to close augeas %s"
msgstr "Nelze zavřít augeas %s"

#, python-format
msgid "Failed to close guest handle %s"
msgstr "Nelze zavřít obslužnou rutinu hosta %s"

msgid ""
"Failed to create block device for instance due to being over volume resource "
"quota"
msgstr ""
"Nelze vytvořit blokové zařízení instance z důvodu překročení kvóty zdrojů "
"svazku"

#, python-format
msgid "Failed to deallocate port %(port_id)s for instance. Error: %(error)s"
msgstr "Nelze zrušit přidělení portu %(port_id)s v instanci. Chyba: %(error)s"

#, python-format
msgid "Failed to delete port %s for instance."
msgstr "Nelze smazat port %s v instanci."

#, python-format
msgid "Failed to delete snapshot from shelved instance (%s)."
msgstr "Nelze smazat snímek z uskladněné instance (%s)."

#, python-format
msgid "Failed to delete volume: %(volume_id)s due to %(exc)s"
msgstr "Nelze smazat svazek: %(volume_id)s kvůli %(exc)s"

#, python-format
msgid "Failed to disassociated floating address: %s"
msgstr "Nelze odloučit plovoucí adresu: %s"

#, python-format
msgid "Failed to extract instance support from %s"
msgstr "Nelze extrahovat podporu instance z %s"

#, python-format
msgid "Failed to get cluster references %s"
msgstr "Nelze získat odkazy clusteru %s"

#, python-format
msgid ""
"Failed to kill a long running process %(pid)s related to the instance when "
"deleting it."
msgstr ""
"Nelze ukončit dlouho běžící proces %(pid)s, související s instancí, při "
"jejím mazání."

#, python-format
msgid "Failed to load extension %(ext_factory)s: %(exc)s"
msgstr "Nelze načít rozšíření %(ext_factory)s: %(exc)s"

msgid "Failed to power off instance"
msgstr "Nelze vypnout instanci"

msgid "Failed to resume instance"
msgstr "Nelze obnovit instanci"

#, python-format
msgid "Failed to revert task state for instance. Error: %s"
msgstr "Nelze vrátit stav úkolu instance. Chyba: %s"

#, python-format
msgid "Failed to rollback quota for deallocate fixed IP: %s"
msgstr "Nelze vrátit zpět kvótu pro zrušení přidělení pevných IP adres: %s"

#, python-format
msgid "Failed to shutdown appliance %s"
msgstr "Nelze vypnout zařízení %s"

msgid "Failed to soft reboot instance. Trying hard reboot."
msgstr "Nelze měkce restartovat instanci. Zkouší se tvrdý restart."

#, python-format
msgid ""
"Fast cloning is only supported on default local SR of type ext. SR on this "
"system was found to be of type %s. Ignoring the cow flag."
msgstr ""
"Rychlé klonování je podporováno pouze na výchozím SR typu ext. SR na tomto "
"systému má typ %s. Ignorování příznaku cow."

msgid "File injection into a boot from volume instance is not supported"
msgstr "Vkládání souboru do zavedení z instance svazku není podporováno"

#, python-format
msgid "Flag sr_matching_filter '%s' does not respect formatting convention"
msgstr "Příznak sr_matching_filter '%s' se neřídí pravidly formátování"

msgid "Flavor could not be found, skipping migration."
msgstr "Konfigurace nenalezena, přesunutí přeskočeno."

#, python-format
msgid ""
"Floating IP address |%(address)s| no longer belongs to instance "
"%(instance_uuid)s. Will not migrate it "
msgstr ""
"Plovoucí IP adresa |%(address)s| již nepatří instanci %(instance_uuid)s. "
"Nebude přesunuta."

#, python-format
msgid ""
"Floating IP address |%(address)s| no longer belongs to instance "
"%(instance_uuid)s. Will not setup it."
msgstr ""
"Plovoucí IP adresa |%(address)s| již nepatří instanci %(instance_uuid)s. "
"Nebude nastavena."

#, python-format
msgid ""
"Found multiple matches for domain %(domain)s.\n"
"%(entry)s"
msgstr ""
"Nalezeno mnoho shod pro doménu %(domain)s.\n"
"%(entry)s"

#, python-format
msgid "Get esx cookies failed: %s"
msgstr "Získání souborů cookies esx selhalo: %s"

#, python-format
msgid ""
"Host %(hostname)s has more disk space than database expected (%(physical)s "
"GB > %(database)s GB)"
msgstr ""
"Hostitel %(hostname)s má větší kapacitu disku, než databáze očekávala "
"(%(physical)s GB > %(database)s GB)"

msgid ""
"Host field should not be set on the instance until resources have been "
"claimed."
msgstr ""
"Pole hostitele by nemělo být nastaveno pro instanci, dokud nebudou získány "
"zdroje."

#, python-format
msgid ""
"Hostname %(hostname)s is longer than 63, truncate it to %(truncated_name)s"
msgstr ""
"Název hostitele %(hostname)s je delší než 63 znaků, zkrácen na "
"%(truncated_name)s"

msgid "Hypervisor driver does not support resume guests"
msgstr "Ovladač hypervizoru nepodporuje obnovení hostů"

#, python-format
msgid "IP %s leased that is not associated"
msgstr "Vypůjčená IP %s není přidružena"

#, python-format
msgid "IP %s released that is not associated"
msgstr "Uvolněna IP %s, která není přidružena"

#, python-format
msgid "IP %s released that was not leased"
msgstr "Uvolněna IP %s, která nebyla vypůjčena"

#, python-format
msgid "IP |%s| leased that isn't allocated"
msgstr "Vypůjčena IP |%s|, která není přidělena"

#, python-format
msgid "ISO creation tool '%s' does not exist."
msgstr "Nástroj pro vytvoření ISO '%s' neexistuje."

#, python-format
msgid ""
"Ignoring DiskNotFound exception while detaching volume %(volume_id)s from "
"%(mp)s: %(err)s"
msgstr ""
"Ignorován í výjimky Disk nenalezen, při odpojování svazku %(volume_id)s z "
"%(mp)s: %(err)s"

#, python-format
msgid "Ignoring Volume Error on vol %(vol_id)s during delete %(exc)s"
msgstr "Ignorování chyby svazku %(vol_id)s během mazání %(exc)s"

#, python-format
msgid "Ignoring error injecting %(inject)s into image (%(e)s)"
msgstr "Ignorování chyby při vkládání %(inject)s do obrazu (%(e)s)"

#, python-format
msgid "Ignoring error injecting data into image %(image)s (%(e)s)"
msgstr "Ignorování chyby při vkládání dat do obrazu %(image)s (%(e)s)"

#, python-format
msgid "Ignoring error while configuring instance with agent: %s"
msgstr "Ignorování chyby při konfiguraci instance pomocí agenta: %s"

#, python-format
msgid "Ignoring exception %(exc)s when getting PBDs for %(sr_ref)s"
msgstr "Ignorování výjimky %(exc)s při získávání PBD pro %(sr_ref)s"

#, python-format
msgid "Ignoring exception %(exc)s when unplugging PBD %(pbd)s"
msgstr "Ignorování výjimky %(exc)s při odpojování PBD %(pbd)s"

#, python-format
msgid "Ignoring malformed capability '%s'. Format should be 'key:val'."
msgstr "Ignorování poškozené schopnosti '%s'. Formát by měl být 'key:val'."

#, python-format
msgid ""
"Ignoring supplied device name: %(device_name)s. Libvirt can't honour user-"
"supplied dev names"
msgstr ""
"Ignorování zadaného názvu zařízení: %(device_name)s. Libvirt nemůže ctít "
"uživatelem zadané názvy dev"

#, python-format
msgid "Ignoring supplied device name: %(suggested_dev)s"
msgstr "Ignorování zadaného názvu zařízení: %(suggested_dev)s"

#, python-format
msgid "Ignoring the invalid elements of the option %(name)s: %(options)s"
msgstr "Ignorování neplatných prvků volby %(name)s: %(options)s"

#, python-format
msgid "Ignoring volume cleanup failure due to %s"
msgstr "Ignorování selhání čištění svazku kvůli %s"

msgid "Image not found during snapshot"
msgstr "Obraz nenalezen během snímkování"

msgid ""
"In vmwareapi:vmops:_destroy_instance, exception while deleting the VM "
"contents from the disk"
msgstr ""
"V vmwareapi:vmops:_destroy_instance, obdržena výjimka při mazání obsahu VM z "
"disku"

#, python-format
msgid ""
"In vmwareapi:vmops:_destroy_instance, got this exception while un-"
"registering the VM: %s"
msgstr ""
"V vmwareapi:vmops:_destroy_instance, obdržena tato výjimka při rušení "
"registrace VM: %s"

msgid "Info cache for instance could not be found. Ignore."
msgstr "Mezipaměť informaci instance nemohla být nalezena. Ignorováno."

#, python-format
msgid ""
"Instance %(instance)s is using a backing file %(backing)s which does not "
"appear in the image service"
msgstr ""
"Instance %(instance)s používá soubor zálohy %(backing)s, který se neobjevuje "
"ve službě obrazu"

#, python-format
msgid ""
"Instance %(uuid)s appears to not be owned by this host, but by %(host)s. "
"Startup processing is being skipped."
msgstr ""
"Instance %(uuid)s se nezdá být ve vlastnictví tohoto hostitele, ale "
"hostitele %(host)s. Zpracování při spuštění je přeskočeno."

msgid "Instance build timed out. Set to error state."
msgstr "Při sestavování instance vypršel čas. Nastavena na chybový stav."

msgid "Instance disappeared during reboot"
msgstr "Instance během restartování zmizela"

msgid "Instance disappeared while detaching a PCI device from it."
msgstr "Instance zmizela během odpojování jejího zařízení PCI."

msgid "Instance does not exist on backend"
msgstr "Instance v této podpůrné vrstvě neexistuje"

msgid ""
"Instance does not exists. Proceeding to delete instance properties on "
"datastore"
msgstr ""
"Instance neexistuje. Pokračuje se v mazání vlastností instance na datovém "
"úložišti"

#, python-format
msgid "Instance failed network setup (attempt %(attempt)d of %(attempts)d)"
msgstr "Instance nemohla nastavit síť (pokus %(attempt)d z %(attempts)d)"

msgid "Instance has had its instance_type removed from the DB"
msgstr "Instance měla svůj typ odebrán z DB"

#, python-format
msgid ""
"Instance in transitional state (%(task_state)s) at start-up and power state "
"is (%(power_state)s), clearing task state"
msgstr ""
"Instance je při spuštění přechodném stavu (%(task_state)s) a stav napájení "
"je (%(power_state)s), stav úkolu je vyčištěn"

#, python-format
msgid ""
"Instance in transitional state (%(task_state)s) at start-up and power state "
"is (%(power_state)s), clearing task state and unpausing the instance"
msgstr ""
"Instance je při spuštění přechodném stavu (%(task_state)s) a stav napájení "
"je (%(power_state)s), stav úkolu je vyčištěn a pozastavení instance zrušeno"

msgid "Instance is not (soft-)deleted."
msgstr "Instance není (měkce) smazána."

#, python-format
msgid ""
"Instance is not stopped. Calling the stop API. Current vm_state: "
"%(vm_state)s, current task_state: %(task_state)s, original DB power_state: "
"%(db_power_state)s, current VM power_state: %(vm_power_state)s"
msgstr ""
"Instance není zastavena. Voláno zastavení API. Současný stav vm: "
"%(vm_state)s, současný stav úkolu: %(task_state)s, původní stav napájení v "
"DB: %(db_power_state)s, současný stav napájení VM: %(vm_power_state)s"

msgid "Instance is paused unexpectedly. Ignore."
msgstr "Instance neočekávaně pozastavena. Ignorována."

msgid "Instance is suspended unexpectedly. Calling the stop API."
msgstr "Instance byla neočekávaně uspána. Volání zastavení API."

msgid "Instance is unexpectedly not found. Ignore."
msgstr "Instance nemohla být neočekávaně nalezena. Ignorována"

msgid ""
"Instance lifecycle events from the compute driver have been disabled. Note "
"that lifecycle changes to an instance outside of the compute service will "
"not be synchronized automatically since the _sync_power_states periodic task "
"is also disabled."
msgstr ""
"Události životního cyklu instance z ovladače výpočtu byly zakázány. "
"Nezapomeňte, že změny životního cyklu instance mimo službu výpočtu nebudou "
"automaticky synchronizovány protože pravidelný úkol _sync_power_states je "
"také zakázán."

msgid "Instance may be still running, destroy it again."
msgstr "Instance může stále běžet, bude znovu zničena."

msgid "Instance not resizing, skipping migration."
msgstr "Velikost instance nebude změněna, přesunutí přeskočeno."

#, python-format
msgid ""
"Instance shutdown by itself. Calling the stop API. Current vm_state: "
"%(vm_state)s, current task_state: %(task_state)s, original DB power_state: "
"%(db_power_state)s, current VM power_state: %(vm_power_state)s"
msgstr ""
"Instance se sama vypnula. Voláno zastavení API. Současný stav vm: "
"%(vm_state)s, současný stav úkolu: %(task_state)s, původní stav napájení v "
"DB: %(db_power_state)s, současný stav napájení VM: %(vm_power_state)s"

#, python-format
msgid ""
"Instance uuid could not be retrieved for instance %s. Instance state change "
"event will be ignored."
msgstr ""
"UUID instance nemohlo být získáno pro instanci %s. Událost změny stavu "
"instance bude ignorována."

#, python-format
msgid "IntegrityError detected when archiving table %(tablename)s: %(error)s"
msgstr ""
"Při archivaci tabulky %(tablename)s došlo k chybě celistvosti: %(error)s"

msgid "Invalid 'agent_present' value. Falling back to the default."
msgstr "Neplatná hodnota 'agent_present'. Vrací se zpět na výchozí."

#, python-format
msgid "Invalid cachemode %(cache_mode)s specified for disk type %(disk_type)s."
msgstr ""
"Zadán neplatný režim ukládaní do mezipaměti %(cache_mode)s pro disk "
"%(disk_type)s."

#, python-format
msgid "Invalid floating IP ID %s in request"
msgstr "Neplatné ID plovoucí IP adresy %s v žádosti"

#, python-format
msgid "Invalid value '%d' for image_compression_level"
msgstr "Neplatná hodnota '%d' pro image_compression_level"

#, python-format
msgid "Invalid value '%s' for torrent_images"
msgstr "Neplatná hodnota '%s' pro torrent_images"

#, python-format
msgid "Invalid value for id mapping %s"
msgstr "Neplatná hodnota pro mapování id %s"

#, python-format
msgid "Key '%(key)s' not in instances '%(inst)s'"
msgstr "Klíč '%(key)s' není v instancích '%(inst)s'"

msgid ""
"Libvirt module could not be loaded. NWFilterFirewall will not work correctly."
msgstr ""
"Modul libvirt nemohl být nalezen. NWFilterFirewall nebude správně fungovat."

#, python-format
msgid "Live migration not completed after %d sec"
msgstr "Přesun za provozu nebyl dokončen po %d vteřinách"

#, python-format
msgid "Live migration stuck for %d sec"
msgstr "Přesun za provozu je zaseknut již %d vteřin"

#, python-format
msgid ""
"Lost connection to XenAPI during call to %(plugin)s.%(fn)s.  Retrying call."
msgstr ""
"Připojení k XenAPI selhalo během volání %(plugin)s.%(fn)s. Nový pokus o "
"volání."

msgid "Lost connection to memcache server for reporting service status."
msgstr ""
"Připojení k serveru memcache ztraceno, stav služby nemůže být kontrolován."

msgid "Lost connection to nova-conductor for reporting service status."
msgstr ""
"Připojení k nová-conductor ztraceno. Stav služby nemůže být kontrolován."

#, python-format
msgid ""
"Metadata value %(value)s for %(key)s is not of type %(type)s. Using default "
"value %(default)s."
msgstr ""
"Hodnota popisných dat %(value)s pro %(key)s není typu %(type)s. Použije se "
"výchozí hodnota %(default)s."

#, python-format
msgid "Migration %s is not found."
msgstr "Přesun %s nenalezen."

msgid "Migration operation was cancelled"
msgstr "Operace přesunu byla zrušena"

#, python-format
msgid "Mountpoint cannot be translated: %s"
msgstr "Bod připojení nelze přeložit: %s"

#, python-format
msgid "Multiple base images for image: %s"
msgstr "Mnoho základních obrazů pro obraz: %s"

#, python-format
msgid ""
"Network %(id)s not matched with the tenants network! The ports tenant "
"%(tenant_id)s will be used."
msgstr ""
"Síť %(id)s se neshoduje se současnou sítí nájemníka! Budou použity porty "
"nájemníka %(tenant_id)s."

#, python-format
msgid ""
"Neutron error: %(ip)s is not a valid IP address for network %(network_id)s."
msgstr "Chyba Neutron: %(ip)s není platnou IP adresou pro síť %(network_id)s."

#, python-format
msgid "Neutron error: Fixed IP %s is already in use."
msgstr "Chyba Neutron: Pevná IP adresa %s se již používá."

#, python-format
msgid ""
"Neutron error: MAC address %(mac)s is already in use on network %(network)s."
msgstr "Chyba Neutron: MAC adresa %(mac)s se již používá v síti %(network)s."

#, python-format
msgid "Neutron error: No more fixed IPs in network: %s"
msgstr "Chyba Neutron: Žádné další pevné IP adresy v síti: %s"

#, python-format
msgid "Neutron error: Port quota exceeded in tenant: %s"
msgstr "Chyba Neutron: Kvóta portů překročena u nájemníka: %s"

#, python-format
msgid "No Network for FixedIP: %s"
msgstr "Žádná síť pro pevnou ip adresu: %s"

#, python-format
msgid "No VirtualInterface for FixedIP: %s"
msgstr "Žádné virtuální rozhraní pro pevnou ip adresu: %s"

#, python-format
msgid "No cell_name for %(method)s() from API"
msgstr "Žádný název buňky pro %(method)s() z API"

msgid "No cell_name for instance update from API"
msgstr "Žádný název buňky pro aktualizaci instance z API"

#, python-format
msgid "No compute node record for %(host)s:%(node)s"
msgstr "nenalezen žádný výpočetní uzel pro %(host)s:%(node)s"

#, python-format
msgid "No compute service record found for host %(host)s"
msgstr "U hostitele %(host)s nenalezen žádný záznam o výpočtové službě"

msgid "No free nbd devices"
msgstr "Žádná volná zařízení nbd"

#, python-format
msgid ""
"No host name specified for the notification of HostAPI.%s and it will be "
"ignored"
msgstr "Pro oznámení HostAPI.%s nezadán žádný název hostitele, bude ignorováno"

#, python-format
msgid "No match when trying to update BDM: %(bdm)s"
msgstr "Žádná shoda při pokusu o aktualizaci BDM: %(bdm)s"

msgid "No more network or fixed IP to be allocated"
msgstr "Žádné další sítě nebo pevné IP adresy k přidělení"

#, python-format
msgid ""
"No protocol specified in for api_server '%s', please update [glance] "
"api_servers with fully qualified url including scheme (http / https)"
msgstr ""
"U api serveru '%s' nebyl zadán žádný protokol, prosím změňte URL u [glance] "
"api serverů na úplnou URL včetně schématu (http / https)"

msgid "No valid host found for rebuild"
msgstr "Nenalezen žádný platný hostitel pro znovu sestavení"

msgid "No valid host found for unshelve instance"
msgstr "Nenalezen žádný platný hostitel pro vyskladnění instance"

#, python-format
msgid ""
"Node %(uuid)s has a malformed \"%(prop)s\". It should be an integer but its "
"value is \"%(value)s\"."
msgstr ""
"Uzel %(uuid)s má poškozené \"%(prop)s\". Mělo by být celé číslo, ale jeho "
"hodnota je \"%(value)s\"."

#, python-format
msgid "Node %(uuid)s has a malformed \"%(prop)s\". It should be an integer."
msgstr "Uzel %(uuid)s má poškozené \"%(prop)s\". Mělo by být celé číslo."

msgid ""
"Node field should not be set on the instance until resources have been "
"claimed."
msgstr ""
"Pole uzel by nemělo být nastaveno pro instanci, dokud nebudou získány zdroje."

#, python-format
msgid "Not deleting key %s"
msgstr "Klíč %s nebude smazán"

msgid ""
"Old property name \"hw_watchdog_action\" is now deprecated and will be "
"removed in the next release. Use updated property name \"hw:watchdog_action"
"\" instead"
msgstr ""
"Název vlastnosti \"hw_watchdog_action\" je nyní zastaralý a bude odstraněn v "
"další verzi. Místo toho používejte aktualizovaný název vlastnosti \"hw:"
"watchdog_action\""

msgid "Paused instance shutdown by itself. Calling the stop API."
msgstr "Pozastavená instance se sama vypnula. Voláno zastavení API."

#, python-format
msgid "Periodic cleanup failed to delete instance: %s"
msgstr "Pravidelné čištění nemohlo smazat instanci: %s"

#, python-format
msgid "Periodic reclaim failed to delete instance: %s"
msgstr "Pravidelné znovu získání nemohlo smazat instanci: %s"

#, python-format
msgid ""
"Periodic task is updating the host stat, it is trying to get disk "
"%(i_name)s, but access is denied. It is most likely due to a VM that exists "
"on the compute node but is not managed by Nova."
msgstr ""
"Pravidelný úkol aktualizuje statistiky hostitele, snaží se získat disk "
"%(i_name)s, ale přístup byl zamítnut. Důvodem je pravděpodobně Vm, která "
"existuje na výpočetním uzlu, ale Nova ho nespravuje."

#, python-format
msgid ""
"Periodic task is updating the host stat, it is trying to get disk "
"%(i_name)s, but disk file was removed by concurrent operations such as "
"resize."
msgstr ""
"Pravidelný úkol aktualizuje statistiky hostitele, snaží se získat disk "
"%(i_name)s, ale souboru disku byl odstraněn souběžnými operacemi jako "
"například změna velikosti."

#, python-format
msgid ""
"Periodic task is updating the host stats, it is trying to get disk info for "
"%(i_name)s, but the backing volume block device was removed by concurrent "
"operations such as resize. Error: %(error)s"
msgstr ""
"Pravidelný úkol aktualizuje statistiky hostitele, snaží se získat informace "
"o disku %(i_name)s, ale záložní zařízení bloku svazku bylo odstraněno "
"souběžnými operacemi jako například změna velikosti. Chyba: %(error)s"

msgid ""
"Please consider to execute setup before trying to configure debug log "
"message."
msgstr ""
"Prosím zvažte spustit instalaci předtím, než se pokusíte nastavit zprávu "
"záznamu ladění."

#, python-format
msgid "Port %s does not exist"
msgstr "Port %s neexistuje"

#, python-format
msgid "Power state %s not found."
msgstr "Stav napájení %s nebyl nalezen."

#, python-format
msgid "Quota exceeded for %s, tried to allocate floating IP"
msgstr "Překročena kvóta pro %s, pokus o přidělení plovoucí IP adresy"

#, python-format
msgid ""
"Quota exceeded for project %(pid)s, tried to allocate fixed IP. %(used)s of "
"%(allowed)s are in use or are already reserved."
msgstr ""
"Kvóta překročena pro projekt %(pid)s, pokus o přidělení pevné ip adresy. "
"%(used)s z %(allowed)s se již používají nebo jsou vyhrazeny."

msgid "Race detected in libvirt.virDomain.info, trying one more time"
msgstr ""
"Zjištěn souběh v libvirt.virDomain.info, bude proveden ještě jeden pokus"

msgid "Reboot failed but instance is running"
msgstr "Restart selhal ale instance běží"

msgid ""
"Reboot failed due to bad volumes, detaching bad volumes and starting halted "
"instance"
msgstr ""
"Restart selhal kvůli špatným svazkům, špatné svazky budou odpojeny a "
"pozastavená instance bude spuštěna"

msgid "Rejecting snapshot request, snapshots currently disabled"
msgstr "Odmítnutí žádosti o snímek, snímky jsou nyní zakázány"

msgid ""
"Relative blockrebase support was not detected. Continuing with old behaviour."
msgstr ""
"Podpora relativního přeskládání bloků nebyla zjištěna. Pokračuje se za "
"použití starého chování."

#, python-format
msgid "Removed %(num)d duplicate rules for floating IP %(float)s"
msgstr ""
"Odstraněno %(num)d duplicitních pravidel pro plovoucí IP adresy %(float)s"

#, python-format
msgid ""
"Report interval must be less than service down time. Current config: "
"<service_down_time: %(service_down_time)s, report_interval: "
"%(report_interval)s>. Setting service_down_time to: %(new_service_down_time)s"
msgstr ""
"Interval hlášení musí být menší než doba nečinnosti. Současné nastavení "
"<doba nečinnosti služby: %(service_down_time)s, interval hlášení "
"%(report_interval)s>. Nastavování doby nečinnosti služby na: "
"%(new_service_down_time)s"

#, python-format
msgid "Retry SR scan due to error: %s"
msgstr "Hledání SR bude znovu provedeno kvůli chybě: %s"

#, python-format
msgid ""
"Retrying %(name)s after a MessagingTimeout, attempt %(attempt)s of "
"%(retries)s."
msgstr ""
"Další pokus %(name)s po vypršení limitu zprávy, pokus %(attempt)s z "
"%(retries)s."

#, python-format
msgid "Root disk file creation failed - %s"
msgstr "Vytvoření souboru kořenového disku selhalo - %s"

#, python-format
msgid ""
"Running Nova with a VMware vCenter version less than %(version)s is "
"deprecated. The required minimum version of vCenter will be raised to "
"%(version)s in the 13.0.0 release."
msgstr ""
"Spouštět Nova pomocí VMware vCenter s verzí starší než %(version)s je "
"zastaralé. Požadovaná minimální verze vCenter bude ve verzi 13.0.0 zvýšena "
"na %(version)s."

msgid ""
"Running libvirt-lxc without user namespaces is dangerous. Containers spawned "
"by Nova will be run as the host's root user. It is highly suggested that "
"user namespaces be used in a public or multi-tenant environment."
msgstr ""
"Spouštění libvirt-lxc bez jmenných prostorů uživatele je nebezpečné. "
"Kontejnery vytvořené Nova budou spuštěny jako kořenový uživatel hostitele. "
"Ve veřejných prostředích, či prostředí mající mnoho nájemníků se velmi "
"doporučuje použít jmenné prostory uživatelů."

#, python-format
msgid ""
"Selected host: %(host)s failed to consume from instance. Error: %(error)s"
msgstr ""
"Zvolený hostitel: %(host)s nemohl spotřebovat z instance. Chyba: %(error)s"

msgid "Server with unsupported policy cannot be rebuilt"
msgstr "Server s nepodporovanou zásadou nemůže být znovu sestaven"

msgid "Service killed that has no database entry"
msgstr "Ukončena služba bez záznamu v databázi"

#, python-format
msgid "Setting instance to %s state."
msgstr "Nastavování instance na stav %s."

#, python-format
msgid "Setting migration %(migration_id)s to error: %(reason)s"
msgstr "Nastavování přesunu %(migration_id)s na chybu: %(reason)s"

#, python-format
msgid "Skipping detach because VBD for %s was not found"
msgstr "Přeskakování odpojení  protože VBD pro %s nebylo nalezeno"

#, python-format
msgid ""
"Subnet(s) too large, defaulting to /%s.  To override, specify network_size "
"flag."
msgstr ""
"Podsítě jsou příliš velké, bude použito /%s. Pro potlačení zadejte příznak "
"network_size."

#, python-format
msgid ""
"Tenant_id %(tenant_id)s does not match tenant_id of instance %(instance_id)s."
msgstr ""
"ID nájemníka %(tenant_id)s se neshoduje s id nájemníka v instanci "
"%(instance_id)s."

msgid ""
"The TrustedFilter is considered experimental by the OpenStack project "
"because it receives much less testing than the rest of Nova. This may change "
"in the future, but current deployers should be aware that the use of it in "
"production right now may be risky."
msgstr ""
"Funkce důvěryhodného filtru v Nova je v projektu OpenStack považována za "
"experimentální, protože prošla menším počtem zkoušek v porovnání s ostatní "
"mi části Nova. Toto se v budoucnu může změnit, ale v současnosti by si "
"správci měli být vědomi, že použití této funkce v ostrých provozech může být "
"riskantní."

msgid ""
"The cells feature of Nova is considered experimental by the OpenStack "
"project because it receives much less testing than the rest of Nova. This "
"may change in the future, but current deployers should be aware that the use "
"of it in production right now may be risky. Also note that cells does not "
"currently support rolling upgrades, it is assumed that cells deployments are "
"upgraded lockstep so n-1 cells compatibility does not work."
msgstr ""
"Funkce buněk v Nova je v projektu OpenStack považována za experimentální, "
"protože prošla menším počtem zkoušek v porovnání s ostatní mi části Nova. "
"Toto se v budoucnu může změnit, ale v současnosti by si správci měli být "
"vědomi, že použití této funkce v ostrých provozech může být riskantní. Také "
"nezapomeňte, že buňky v současnosti nepodporují postupné aktualizace, "
"předpokládá se, že při jejich nasazení budou aktualizovány synchronizovaně, "
"takže kompatibilita buněk n-1 nefunguje."

#, python-format
msgid ""
"The same keypair name '%(name)s' exists in the following cells: "
"%(cell_names)s. The keypair value from the first cell is returned."
msgstr ""
"Stejný název páru klíčů '%(name)s' existuje v následujících buňkách: "
"%(cell_names)s. Je vrácena hodnota páru z první buňky."

#, python-format
msgid "The volume %s does not exist!"
msgstr "Svazek %s neexistuje!"

msgid ""
"This key manager is insecure and is not recommended for production "
"deployments"
msgstr ""
"Tento správce klíčů není bezpečný a nedoporučuje se ho používat v ostrých "
"provozech"

msgid "This shouldn't be getting called except during testing."
msgstr "Toto by nemělo být voláno kromě případů testování."

msgid ""
"Timed out waiting for nova-conductor.  Is it running? Or did this service "
"start before nova-conductor?  Reattempting establishment of nova-conductor "
"connection..."
msgstr ""
"Vypršel čas čekání na nova-conductor. Je spuštěn? Nebo byla tato služba "
"spuštěna před nova-conductor? Nový pokus o navázání spojení s nova-"
"conductor..."

msgid "Timed out while waiting for soft shutdown."
msgstr "Při čekání na měkké vypnutí vypršel čas."

#, python-format
msgid "Timeout waiting for vif plugging callback for instance %(uuid)s"
msgstr ""
"Vypršel čas při čekání na zpětné volání zapojení virtuálního rozhraní pro "
"instanci %(uuid)s"

msgid "Too many id maps, only included first five."
msgstr "Příliš mnoho map id, zahrnuto pouze prvních pět."

#, python-format
msgid ""
"Treating negative config value (%(retries)s) for 'block_device_retries' as 0."
msgstr ""
"Záporná hodnota nastavení (%(retries)s) v 'block_device_retries' je "
"vyhodnocena jako 0."

#, python-format
msgid ""
"Treating negative config value (%(retries)s) for 'glance.num_retries' as 0."
msgstr ""
"Záporná hodnota nastavení (%(retries)s) v 'glance.num_retries' je "
"vyhodnocena jako 0."

#, python-format
msgid ""
"Tried to remove rule that was not there: %(chain)r %(rule)r %(wrap)r %(top)r"
msgstr ""
"Pokus o odstranění neexistujícího pravidla: %(chain)r %(rule)r %(wrap)r "
"%(top)r"

#, python-format
msgid ""
"Trying to remove device with %(status)s ownership %(instance_uuid)s because "
"of %(pci_exception)s"
msgstr ""
"Pokus o odstranění zařízení se stavem %(status)s ve vlastnictví "
"%(instance_uuid)s z důvodu %(pci_exception)s"

#, python-format
msgid "URI %(uri)s does not support connection events: %(error)s"
msgstr "URI %(uri)s nepodporuje události připojení: %(error)s"

#, python-format
msgid "URI %(uri)s does not support events: %(error)s"
msgstr "URI %(uri)s nepodporuje události: %(error)s"

#, python-format
msgid "URI %(uri)s does not support full set of host capabilities: %(error)s"
msgstr "URI %(uri)s nepodporuje úplnou sadu schopností hostitele: %(error)s"

#, python-format
msgid "URI %(uri)s does not support listDevices: %(error)s"
msgstr "URI %(uri)s nepodporuje výpis zařízení: %(error)s"

#, python-format
msgid "Unable to delete %(file)s. Exception: %(ex)s"
msgstr "Nelze smazat %(file)s. Výjimka: %(ex)s"

#, python-format
msgid "Unable to dequalify.  %(name)s is not in %(domain)s.\n"
msgstr "Nelze dekvalifikovat.  %(name)s není v %(domain)s.\n"

msgid ""
"Unable to find a different image to use for rescue VM, using instance's "
"current image"
msgstr ""
"Nelze najít jiný obraz použitelný pro záchranu VM, bude použit současný "
"obraz instance"

#, python-format
msgid ""
"Unable to find network matching '%(network_name)s', user will have to enter "
"IP manually..."
msgstr ""
"Nelze najít síť odpovídající '%(network_name)s', uživatel bude síť muset "
"zadat ručně..."

#, python-format
msgid "Unable to force TCG mode, libguestfs too old? %s"
msgstr "Nelze vynutit režim TCG, libguestfs je příliš stary? %s"

#, python-format
msgid "Unable to increase max downtime to %(time)dms: %(e)s"
msgstr "Nelze zvýšit maximální čas odstávky na %(time)dms: %(e)s"

#, python-format
msgid "Unable to mount image %(image)s with error %(error)s. Cannot resize."
msgstr ""
"Nelze připojit obraz %(image)s s chybou %(error)s. Nelze změnit velikost."

#, python-format
msgid "Unable to stat %(disk)s: %(ex)s"
msgstr "Nelze nalézt %(disk)s: %(ex)s"

msgid "Unable to update metadata, VM not found."
msgstr "Nelze aktualizovat popisná data, VM nenalezeno."

#, python-format
msgid "Unable to update the agent due to: %(exc)s"
msgstr "Nelze aktualizovat agenta kvůli: %(exc)s"

#, python-format
msgid ""
"Unexpected confirmation status '%(status)s' of migration %(id)s, exit "
"confirmation process"
msgstr ""
"Neočekávaný stav potvrzení '%(status)s' v přesunu %(id)s, potvrzovací proces "
"ukončen"

#, python-format
msgid "Unexpected migration job type: %d"
msgstr "Neočekávaný typ úkolu přesunu: %d"

#, python-format
msgid "Unexpected power state %d"
msgstr "Neočekávaný stav napájení %d"

#, python-format
msgid "Unknown base file: %s"
msgstr "Neznámý základní soubor: %s"

#, python-format
msgid "Unknown content in connection_info/qos_specs: %s"
msgstr "Neznámý obsah v informaci o připojení/specifikaci qos: %s"

#, python-format
msgid "Unrecognized cache_images value '%s', defaulting to True"
msgstr "Nerozpoznaná hodnota cache_images '%s', zvolena výchozí hodnota True"

msgid "VCPUs not set; assuming CPU collection broken"
msgstr "VCPU nenastaveny; předpokládáno poškození sbírky CPU"

msgid "VM already halted, skipping shutdown..."
msgstr "VM již je zastaven, přeskakování vypnutí..."

msgid "VM is not present, skipping destroy..."
msgstr "VM není přítomno, přeskakování ničení..."

msgid "VM is not present, skipping soft delete..."
msgstr "VM není přítomno, přeskakování měkkého smazání..."

#, python-format
msgid ""
"VMware vCenter version less than %(version)s does not support the 'ovs' port "
"type."
msgstr ""
" VMware vCenter s verzí starší než %(version)s nepodporuje typ portu 'ovs'."

msgid ""
"Value specified in conf file for xenserver.torrent_base_url does not contain "
"a slash character, therefore it will not be used as part of the torrent URL. "
"Specify a valid base URL as defined by RFC 1808 (see step 6)."
msgstr ""
"Hodnota zadaná v souboru s nastavením pro xenserver.torrent_base_url "
"neobsahuje znak závorky a proto nebude použita jako součást URl torrentu. "
"Zadejte platnou základní URL tak jak jí určuje RFC 1808 (viz krok 6)."

#, python-format
msgid "Virtual disk: %s of volume's backing not found."
msgstr "Virtuální disk: %s nenalezeno v záloze svazku ."

#, python-format
msgid "Volume %s should be encrypted but there is no encryption metadata."
msgstr ""
"Svazek %s by měl být zašifrován, ale neexistují žádná popisná data šifrování."

#, python-format
msgid ""
"Volume group %(vg)s will not be able to hold sparse volume %(lv)s. Virtual "
"volume size is %(size)d bytes, but free space on volume group is only "
"%(free_space)db."
msgstr ""
"Skupina svazku %(vg)s nebude moci pojmout řídký svazek %(lv)s. Velikost "
"virtuálního svazku je %(size)d bajtů, ale volná kapacita ve skupině svazku "
"je pouze %(free_space)d bajtů."

#, python-format
msgid ""
"Volume id: %(vol_id)s finished being created but its status is "
"%(vol_status)s."
msgstr ""
"Svazek s id %(vol_id)s dokončil proces vytváření ale jeho stav je "
"%(vol_status)s."

msgid ""
"WARNING: Using the remote debug option changes how Nova uses the eventlet "
"library to support async IO. This could result in failures that do not occur "
"under normal operation. Use at your own risk."
msgstr ""
"VAROVÁNÍ: Používání volby vzdáleného ladění mění jak Nova používá knihovnu "
"eventlet, aby mohla podporovat asynchronní Vstup/Výstup. To může mít za "
"následek selhání, které se při normálních operacích nevyskytují. Používejte "
"na vlastní riziko."

#, python-format
msgid "Warning, memory usage is 0 for %(instance)s on baremetal node %(node)s."
msgstr ""
"Varování, využití paměti %(instance)s je 0 na uzlu bez operačního systému "
"%(node)s."

#, python-format
msgid ""
"Warning, number of cpus is 0 for %(instance)s on baremetal node %(node)s."
msgstr ""
"Varování,počet procesorů %(instance)s je 0 na uzlu bez operačního systému "
"%(node)s."

#, python-format
msgid ""
"While synchronizing instance power states, found %(num_db_instances)s "
"instances in the database and %(num_vm_instances)s instances on the "
"hypervisor."
msgstr ""
"Při synchronizaci stavů napájení instance nalezeno %(num_db_instances)s "
"instancí v databázi a %(num_vm_instances)s v hypervizoru."

msgid ""
"X-Instance-ID present in request headers. The 'service_metadata_proxy' "
"option must be enabled to process this header."
msgstr ""
"V hlavičkách požadavku je přítomno X-Instance-ID. Pro zpracování této "
"hlavičky je třeba mít zapnutu volbu 'service_metadata_proxy'."

#, python-format
msgid ""
"X-Instance-ID-Signature: %(signature)s does not match the expected value: "
"%(expected_signature)s for id: %(requestor_id)s. Request From: "
"%(requestor_address)s"
msgstr ""
"X-Instance-ID-Signature: %(signature)s neodpovídá očekávané hodnotě: "
"%(expected_signature)s pro id: %(requestor_id)s. Formulář žádosti: "
"%(requestor_address)s"

#, python-format
msgid ""
"You are running with libvirt version %s which is known to have broken NUMA "
"support. Consider patching or updating libvirt on this host if you need NUMA "
"support."
msgstr ""
"Máte libvirt verze %s, o niž se ví, že má poškozenou podporu NUMA. Pokud "
"potřebujete podporu NUMA zvažte opravu, nebo aktualizaci libvirt na tomto "
"hostiteli. "

msgid ""
"Your libvirt version does not support the VIR_DOMAIN_XML_MIGRATABLE flag, "
"and the graphics (VNC and/or SPICE) listen addresses on the destination node "
"do not match the addresses on the source node. Since the source node has "
"listen addresses set to either the catch-all address (0.0.0.0 or ::) or the "
"local address (127.0.0.1 or ::1), the live migration will succeed, but the "
"VM will continue to listen on the current addresses."
msgstr ""
"Vaše verze libvirt nepodporuje příznak VIR_DOMAIN_XML_MIGRATABLE a adresy "
"pro odposlech grafiky (VNC a(nebo SPICE) na cílovém uzlu neodpovídají "
"adresám na zdrojovém uzlu. Protože zdrojový uzel má adresu pro "
"odposlouchávání nastavenu buď na vše zachytávající adresu (0.0.0.0 nebo ::), "
"nebo na místní adresu (127.0.0.1 či ::1), přesun za provozu bude úspěšně "
"proveden, ale VM bude stále odposlouchávat na současných adresách."

#, python-format
msgid "_migrate_disk_resizing_up failed to rollback: %s"
msgstr "přesunutí a zvýšení velikosti disku nešlo vrátit zpět: %s"

#, python-format
msgid ""
"attach interface failed , try to deallocate port %(port_id)s, reason: %(msg)s"
msgstr ""
"připojení rozhraní selhalo, zkuste zrušit přidělení portu %(port_id)s, důvod "
"%(msg)s"

#, python-format
msgid "couldn't obtain the memory from domain: %(uuid)s, exception: %(ex)s"
msgstr "nelze získat paměť z domény: %(uuid)s, výjimka: %(ex)s"

#, python-format
msgid ""
"couldn't obtain the vcpu count from domain id: %(uuid)s, exception: %(ex)s"
msgstr "nelze získat počet vcpu z domény s id: %(uuid)s, výjimka: %(ex)s"

#, python-format
msgid "cpu_arch not defined for node '%s'"
msgstr "architektura procesoru by neměla být určena pro uzel '%s'"

msgid "datastore_regex is ignored when PBM is enabled"
msgstr "regulární výraz datového úložiště je ignorován když je PBM zapnuto"

#, python-format
msgid "deallocate port %(port_id)s failed"
msgstr "zrušení přidělení portu %(port_id)s selhalo"

#, python-format
msgid "hostname %s too long, truncating."
msgstr "název hostitele %s je příliš dlouhý, bude zkrácen."

#, python-format
msgid "ignoring missing logical volume %(path)s"
msgstr "ignorování chybějícího logického svazku %(path)s"

#, python-format
msgid ""
"image %(id)s at (%(base_file)s): warning -- an absent base file is in use! "
"instances: %(instance_list)s"
msgstr ""
"Obraz %(id)s v (%(base_file)s): varování -- chybějící základní soubor se "
"používá! Instance: %(instance_list)s"

#, python-format
msgid "image %(volume)s in pool %(pool)s can not be found, failed to remove"
msgstr "Obraz %(volume)s v zásobě %(pool)s nemůže být nalezen, nelze odstranit"

#, python-format
msgid "instance's host %s is down, deleting from database"
msgstr "hostitel instance %s je mimo provoz, probíhá jeho mazání z databáze"

#, python-format
msgid ""
"instance-dns-zone is |%(domain)s|, which is in availability zone |%(zone)s|. "
"Instance is in zone |%(zone2)s|. No DNS record will be created."
msgstr ""
"dns zóna instance je |%(domain)s|, která je v zóně dostupnosti |%(zone)s|. "
"Instance je v zóně |%(zone2)s|. Nebude vytvořen žádný záznam DNS."

#, python-format
msgid "instance-dns-zone not found |%s|."
msgstr "dns zóna instance nenalezena |%s|."

msgid "ipxe_boot is True but no ISO image found"
msgstr "ipxe_boot je nastaveno na True, ale nenalezen žádný obraz ISO"

msgid "ipxe_boot_menu_url not set, user will have to enter URL manually..."
msgstr ""
"ipxe_boot_menu_url není nastaveno, uživatel bude muset URL zadat ručně..."

msgid "ipxe_network_name not set, user will have to enter IP manually..."
msgstr ""
"ipxe_network_name není nastaveno, uživatel bude muset zadat IP ručně..."

#, python-format
msgid "isLuks exited abnormally (status %(exit_code)s): %(stderr)s"
msgstr "isLuks byl mimořádně ukončen (stav %(exit_code)s): %(stderr)s"

#, python-format
msgid "multiple fixed_ips exist, using the first IPv4 fixed_ip: %s"
msgstr ""
"Existuje mnoho pevných ip adres, bude použita první pevná ip adresa IPv4: %s"

#, python-format
msgid ""
"my_ip address (%(my_ip)s) was not found on any of the interfaces: %(ifaces)s"
msgstr ""
"my_ip adresa (%(my_ip)s) nebyla nalezena v žádných rozhraních: %(ifaces)s"

#, python-format
msgid "plug_vifs() failed %(cnt)d. Retry up to %(max_retry)d."
msgstr "plug_vifs() selhalo %(cnt)d. Proces bude zopakován %(max_retry)dkrát."

#, python-format
msgid "rbd remove %(volume)s in pool %(pool)s failed"
msgstr "odstranění rbd %(volume)s v zásobě %(pool)s selhalo"

msgid "set_admin_password is not implemented by this driver or guest instance."
msgstr "set_admin_password není tímto ovladačem nebo hostem instance zavedeno."

#, python-format
msgid ""
"trying to inject a file into a non-running (state: %(current_state)s "
"expected: %(expected_state)s)"
msgstr ""
"pokus o vložení souboru do nespuštěného systému (stav: %(current_state)s "
"očekáváno: %(expected_state)s)"

#, python-format
msgid ""
"trying to reboot a non-running instance: (state: %(state)s expected: "
"%(running)s)"
msgstr ""
"pokus o restartování nespuštěné instance: (stav: %(state)s očekáváno: "
"%(running)s)"

#, python-format
msgid ""
"trying to snapshot a non-running instance: (state: %(state)s expected: "
"%(running)s)"
msgstr ""
"pokus o vytvoření snímku z nespuštěné instance: (stav: %(state)s očekáváno: "
"%(running)s)"

#, python-format
msgid ""
"vmware_ostype from image is not recognised: '%(ostype)s'. An invalid os type "
"may be one cause of this instance creation failure"
msgstr ""
"typ os vmware z obrazu není rozpoznán: '%(ostype)s'. Neplatný typ os může "
"být jednou z příčin selhání vytvoření této instance"
